azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/springcloud/DeployMojo.java

```diff
@@ -152,10 +150,8 @@ protected void doExecute() throws MojoExecutionException, MojoFailureException {
 
     protected boolean confirmDeploy(List<AzureTask<?>> tasks) throws MojoFailureException {
         try (IPrompter prompter = new DefaultPrompter()) {
-            final StringBuilder confirms = new StringBuilder(CONFIRM_PROMPT_START);
-            tasks.stream().filter(t -> StringUtils.isNotBlank(t.getTitle())).forEach((t) ->
-                confirms.append(System.lineSeparator()).append("\t- ").append(t.getTitle()));
-            log.info(confirms.toString());
+            log.info(CONFIRM_PROMPT_START);
```

Comment:
log.info and System.out.printf are not the same, since info can be disabled by set log level to error